README.md

```diff
@@ -0,0 +1,30 @@
+[![Build status](https://badge.buildkite.com/a424560339e282f68e230109b50decf4f7ed5b72afa93e1326.svg?branch=master)](https://buildkite.com/fawkesrobotics/fawkes-build)
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/fawkesrobotics/fawkes.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/fawkesrobotics/fawkes/alerts/)
+[![Language grade: C/C++](https://img.shields.io/lgtm/grade/cpp/g/fawkesrobotics/fawkes.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/fawkesrobotics/fawkes/context:cpp)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/b107df4d77aa46a386c81faf9c425544)](https://www.codacy.com/app/fawkesrobotics/fawkes)
+
+# Fawkes Robot Software Framework
+Fawkes is a component-based Software Framework for Robotic Real-Time Applications for various
+Platforms and Domains.
+
+It is developed and used for cognitive robotics real-time applications like, soccer, domestic
```

Comment:
my last suggestion had two changes, the second one is not included yet:
```suggestion
It is developed and used for cognitive robotics real-time applications such as soccer, domestic
```
(like -> such as)